src/main/java/hudson/remoting/Launcher.java

```diff
@@ -163,8 +165,11 @@ public void addClasspath(String pathList) throws Exception {
             "then wait for the controller to connect to that port.")
     public File tcpPortFile=null;
 
-
-    @Option(name="-auth",metaVar="user:pass",usage="If your Jenkins is security-enabled, specify a valid user name and password.")
+    /**
+     * @deprecated use {@link #agentJnlpCredentials} or {@link #proxyCredentials}
+     */
+    @Deprecated
+    @Option(name="-auth",metaVar="user:pass",usage="(deprecated) unused; use -credentials or -proxyCredentials")
     public String auth = null;
```

Comment:
The implementation of this was dead code, which is now deleted. The only ways of passing credentials that actually did anything were `-credentials` and/or `-proxyCredentials`.